### ISSUE TITLE
Add openvino backend

### ISSUE THREAD

**Issue by daniil-lyakhov** [cipher982/llm-benchmarks#8]
Greetings, @cipher982!

Currently we are working on the Openvino inference framework, and such benchmarks are critical to understand gaps and differences between our framework and Transformers/ TGI / VLLM / llama.cpp frameworks. We have an integration with HF (https://github.com/huggingface/optimum-intel), and I would like to contribute the openvino backend to the benchmark.
 
Is it possible? If so, could you please help me with that? 

First of all, I would like to run the benchmark locally #7 
Thanks!

**Comment by cipher982** [comment]
Yes, I have been meaning to bring the local benchmarks up to a better standard, so I will spend the next few days bring some libraries up to date and improving the experience for others. This was originally designed just for me to automatically run the benchmarks for the frontend webpage and not intended for others to use, but let me go through the code an clean some stuff up.
